testing/botbuilder-test-utils/src/oauth.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
And this the Oauth flow, including fully signed and verifiable JWTs (using the above stub).